mytimer/__main__.py

```diff
@@ -49,7 +51,11 @@ def main():
         nargs="?",
         const=1)
     args = parser.parse_args()
-    run_timer(args)
+    try:
+        run_timer(args)
+    except KeyboardInterrupt:
+        print(MESSAGE_TEMPLATE.format("See you. Bye!"))
+        exit()
```

Comment:
There are differences between `sys.exit()`, `exit()`, `quit()`, and `os._exit()`. Please refer to [here](https://www.geeksforgeeks.org/python-exit-commands-quit-exit-sys-exit-and-os-_exit/).

I think it will be fine also if we avoid calling any functions.